src/Core/src/Handlers/Layout/LayoutHandler.iOS.cs

```diff
@@ -37,6 +38,10 @@ public override void SetVirtualView(IView view)
 			TypedNativeView.CrossPlatformMeasure = VirtualView.Measure;
 			TypedNativeView.CrossPlatformArrange = VirtualView.Arrange;
 
+			//Cleanup the old view when reused
+			var oldChildren = TypedNativeView.Subviews.ToList();
```

Comment:
Should we do this only if HotReload is enabled?

Reply:
No, if we re-use a handler. We need to remove the old view.

Reply:
@hartez  any opinion here?

Reply:
We talked about this in team chat, and it stays for now